docs/self-managed/operational-guides/configure-elasticsearch-privileges.md

```diff
@@ -0,0 +1,33 @@
+---
+id: configure-elasticsearch-privileges
+title: "Configure Elasticsearch privileges"
+---
+
+If you implement Camunda 8 with ElasticSearch as a service provider, you must configure ElasticSearch with the following [privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html) in mind.
+
+## Cluster privileges
+
+To use the [backup feature](/self-managed/operational-guides/backup-restore/backup-and-restore.md), you need to have snapshot privileges:
+
+- `create_snapshot`
+- `monitor_snapshot`
+- `manage_index_templates` to manage index templates
+- `monitor`
+- `manage_ilm` when ILM is enabled
```

Comment:
These three are not required for taking a backup.

* `monitor` is necessary for the health check
* `manage_index_templaces` are kind of to create the index schema on startup (if they don't exist yet in Elasticsearch)
* `manage_ilm` when the index lifecycle management is used to apply a retention policy


I think I would mark the first (monitor) and the second manage_index_templaces privilege as required. The third only when they apply ILM.